docs/api/connection.rst

```diff
@@ -204,6 +204,21 @@ Functions
    :param options: The options object to get the value from
    :returns: The state of the auth plugin option
 
+.. c:function:: void drizzle_options_set_socket_owner(drizzle_options_st *options,
+   drizzle_socket_owner owner)
+
+   Sets the owner of the socket connection
+
+   :param options: The options object to modify
+   :param owner: The owner of the socket connection
+
+.. c:function:: drizzle_socket_owner drizzle_options_get_socket_owner(drizzle_options_st *options)
+
+   Gets the owner of the socket connection
+
+   :param options: The options object to get the value from
+   :returns: The owner of the socket
```

Comment:
You seem to always say "owner of the socket connection", so it seems odd that the word `connection` is missing here.